x-pack/platform/plugins/shared/fleet/server/tasks/unenroll_inactive_agents_task.ts

```diff
@@ -106,6 +107,23 @@ export class UnenrollInactiveAgentsTask {
     return `${TYPE}:${VERSION}`;
   }
 
+  // function marked public to allow testing
+  // find inactive agents enrolled on selected policies
+  // check that the time since last checkin was longer than unenroll_timeout
+  public getAgentsQuery(agentPolicies: AgentPolicy[]): string {
+    const filteredPolicies = agentPolicies.filter(
```

Comment:
I think this filter is not needed, it was already filtered here:
https://github.com/elastic/kibana/blob/2098a0e9d66cee315f35e15b38698fcb9485a093/x-pack/platform/plugins/shared/fleet/server/tasks/unenroll_inactive_agents_task.ts#L140